src/Raven.Server/Utils/ThreadsUsage.cs

```diff
@@ -33,7 +33,7 @@ public ThreadsUsage()
                         {
                             // thread has exited
                         }
-                        catch (Win32Exception e) when (e.HResult == 0x5)
+                        catch (Win32Exception e) when (e.NativeErrorCode == 0x5)
```

Comment:
What is the meaning of this?

Reply:
`System.ComponentModel.Win32Exception (**5**): Access is denied.`
I believe he meant to this exception..
(My test is throwing this exception in the second run in ```Tryouts```).

Reply:
And what would this give in Linux?

Reply:
I ran it on Linux and it throws "InvalidOperationException" (which is handled).